### ISSUE TITLE
A problem on update guidance on "tracking_running_stats" setup in EWC algorithm.

### ISSUE THREAD

**Issue by qqish** [Laborieux-Axel/SynapticMetaplasticityBNN#5]
Hi, I have a question on the upadate of 'tracking running stats=False'.
I have set the "track_running_stats=False" in the "batchnorm2d", and got the paper results. But there is another problem when setting that under EWC algorithms. When I run --ewc and --ewc-lambda 5000.0, and set the "track_running_stats=False" , the estimate_fisher function goes wrong. Maybe because the batch size in dset_data_loader is 1? and the error is shown as "except more than 1 value per channel when training, got input size torch.Size([1,4096])" in the line "x=self.layers[self.norm+str(layer+1)](https://github.com/Laborieux-Axel/SynapticMetaplasticityBNN/issues/x)".
So.. what should I do? using the "track_running stats=True" to train the EWC will cause the decay of the first task petformance, but using the "False" one will report the error. Hope that I have explained clearly.

**Comment by Laborieux-Axel** [comment]
Hi,
Thanks for raising this, and sorry that you are having issues reproducing the results.

> using the "track_running stats=True" to train the EWC will cause the decay of the first task petformance

I am assuming you mean that there is the numerical instability also with EWC when `track_running_stats=True`, is it correct? Did you actually try doing that with EWC?

For `track_running_stats=False`, I think you are right that the issue might come from the batch size equal to one for the estimation of the Fisher information, since batchnorm is not defined for one single sample (variance is zero). To be sure whether that is the reason for the error, could you try setting `batch_size=2` in the `estimate_fisher` function, and see if it runs?
